ert/data/record/_transformation.py

```diff
@@ -0,0 +1,59 @@
+from abc import ABC, abstractmethod
+from pathlib import Path
+import tarfile
+import io
+import stat
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ert.data import RecordTransmitter
+
+_BIN_FOLDER = "bin"
+
+
+class RecordTransformation(ABC):
+    @abstractmethod
+    async def dump(
```

Comment:
```suggestion
    async def transform(
```

? 😁

Reply:
I thought about a different function name but wanted to keep the `dump` tradition :smile: Will do the `transform` ...